### PR TITLE
feat: add konva frame and elements

### PR DESCRIPTION
**Pull request by padmajabhol**
Added the konva stage to setup the frame, fixed to toggle switch, added input elements to the frame.

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Added the konva stage to set-up the frame, fixed the toggle switch, added input elements to the frame.
❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: src/components/Frame/Frame.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+const FRAMES = {
+  // eslint-disable-next-line max-len
+  ONE: 'https://res.cloudinary.com/dscnitrourkela/image/upload/icecream_frames/siap9qgjxblbkg1ubmsq.png',
+  // eslint-disable-next-line max-len
+  TWO: 'https://res.cloudinary.com/dscnitrourkela/image/upload/icecream_frames/fvbpjkljamqjaq7kzy7q.png',
+  THREE:
+    // eslint-disable-next-line max-len
+    'https://res.cloudinary.com/dscnitrourkela/image/upload/icecream_frames/p15pic06ielx741wdmh1.png',
+  // eslint-disable-next-line max-len
+  FOUR: 'https://res.cloudinary.com/dscnitrourkela/image/upload/v1610220397/icecream_frames/hd9zlgcsw9zjppn0lw9x.png',
```
Comment (DesignrKnight) [inline src/components/Frame/Frame.jsx:39]: There is an additional `v1610220397` here, Is it needed?

File: src/components/Frame/Frame.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+};
+
+export default function App() {
+  const [selectedFrame, setSelectedFrame] = useState(FRAMES.ONE);
+  const [uploadedImage, setUploadedImage] = useState();
+  const [username, setYourName] = useState('Your Name');
```
Comment (DesignrKnight) [inline src/components/Frame/Frame.jsx:45]: either call it `[username, setUsername]` or `[yourName, setYourName]`

File: src/components/Frame/Frame.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  const [guildname, setGuildname] = useState('Guild Name');
+  const stageRef = useRef(null);
+
+  const [image] = useImage(selectedFrame);
+  const [image2] = useImage(uploadedImage);
```
Comment (DesignrKnight) [inline src/components/Frame/Frame.jsx:50]: Better name instead of `image2` can be done

File: src/components/toolbox/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ImageUpload from './ImageUpload';
+
+const Inputs = ({ name, setName, guildName, setGuildName, uploadedImage, setUploadedImage }) => (
```
Comment (DesignrKnight) [inline src/components/toolbox/Input.jsx:4]: I see that not all props are used, you can ignore the rest if not needed `name, setName, guildName, setGuildName`

File: src/components/Frame/Frame.jsx (hunk 1 above)
Comment (padmajabhol) [reply src/components/Frame/Frame.jsx:39]: No, I'll will remove the versions from all links.

**Review by riteshsp2000 — APPROVED** [review]

**Pull request merged**